### PR TITLE
feat(model): Add optional message field to auditstamp

### PR DESCRIPTION
**Pull request by gabe-lyons**
This allows us to track context around audit stamps.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl [hunk 1]
```diff
@@ -19,4 +19,9 @@ record AuditStamp {
    * The entity (e.g. a service URN) which performs the change on behalf of the Actor and must be authorized to act as the Actor.
    */
   impersonator: optional Urn
-}
\ No newline at end of file
+
+  /**
+   * Additional message to keep track of in the event
```
Comment (shirshanka) [inline li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl:24]: doc string needs to be improved

**Review by shirshanka — CHANGES_REQUESTED** [review]

**Review by jjoyce0510 — APPROVED** [review]
LGTM - Thanks!

**Comment by xiphl** [comment]
This sounds super useful! i can think of the use cases when i programmatically emit updates to datahub already, thanks!

File: li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl (hunk 1 above)
Comment (gabe-lyons) [reply li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl]: @shirshanka updated!

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**